core/src/main/scala/cats/syntax/DistributiveSyntax.scala

```diff
@@ -0,0 +1,14 @@
+package cats
+package syntax
+
+import cats.evidence.===
+
+trait DistributiveSyntax extends Distributive.ToDistributiveOps {
+  implicit final def catsSyntaxDistributiveOps[F[_]: Functor, A](fa: F[A]): DistributiveOps[F, A] = new DistributiveOps[F, A](fa)
```

Comment:
Maybe we should add another class for `cosequence`, we wouldn't need `cats.evidence` then:

```scala
final class CosequenceOps[F[_]: Functor, G[_]: Distributive, A](val fga: F[G[A]]) extends AnyVal {
  def cosequence: G[F[A]] = G.cosequence(fga)
}
```

Reply:
Hmmm, not sure I see a reason why that's better? The evidence constraint is on just the one method and it saves the extra class for syntax. Is there a reason it should be preferred?

Reply:
Maybe you're right, disregard me! :D